src/journal/journalctl.c

```diff
@@ -2392,11 +2392,15 @@ int main(int argc, char *argv[]) {
         /* Opening the fd now means the first sd_journal_wait() will actually wait */
         if (arg_follow) {
                 r = sd_journal_get_fd(j);
-                if (r == -EMEDIUMTYPE) {
+                if (r == -EMFILE) {
+                        log_warning("Insufficent watch descriptors available. Reverting to -n.");
+                        arg_follow = false;
+                }
+                else if (r == -EMEDIUMTYPE) {
```

Comment:
please follow coding style on this one, we generally place `else` right after the previous `}` on the same line in cases like this.